src/data/categories.tsx

```diff
@@ -29,6 +31,12 @@ export const categories = [
   { key: "nfts", name: "NFTs", icon: gallery, iconDark: galleryLight },
   { key: "social", name: "Social", icon: heart, iconDark: heartLight },
   { key: "oracles", name: "Oracles", icon: ball, iconDark: ballLight },
```

Comment:
should this be removed?

Reply:
I was thinking we might need it in future but actually I can't think of many oracles (Chainlink only?) in reality. So yes I'll remove it. We can always add it again if it makes sense to do so.